front/app/containers/Admin/reporting/components/ReportBuilder/Editor/index.tsx

```diff
@@ -1,59 +1,75 @@
 import React from 'react';
-import { SerializedNodes } from '@craftjs/core';
 
 // components
 import { Box } from '@citizenlab/cl2-component-library';
 
 // craft
-import BaseEditor from 'components/admin/ContentBuilder/Editor';
+import { Editor as CraftEditor, SerializedNodes } from '@craftjs/core';
+import RenderNode from './RenderNode';
 import Container from 'components/admin/ContentBuilder/Widgets/Container';
 
 // default widgets
+import Title from 'components/admin/ContentBuilder/Widgets/Title';
 import Text from 'components/admin/ContentBuilder/Widgets/Text';
 import TwoColumn from '../../../components/ReportBuilder/Widgets/TwoColumn';
 import Image from 'components/admin/ContentBuilder/Widgets/Image';
 import WhiteSpace from 'components/admin/ContentBuilder/Widgets/WhiteSpace';
 
-// Report builder widgets
+// report builder widgets
 import AboutReportWidget from '../Widgets/AboutReportWidget';
 import SurveyResultsWidget from '../Widgets/SurveyResultsWidget';
 import VisitorsWidget from '../Widgets/ChartWidgets/VisitorsWidget';
 import VisitorsTrafficSourcesWidget from '../Widgets/ChartWidgets/VisitorsTrafficSourcesWidget';
 import AgeWidget from '../Widgets/ChartWidgets/AgeWidget';
 import GenderWidget from '../Widgets/ChartWidgets/GenderWidget';
 
+// templates
+import ProjectTemplate from '../Templates/ProjectTemplate';
+
 type EditorProps = {
   children?: React.ReactNode;
   isPreview: boolean;
   onNodesChange?: (nodes: SerializedNodes) => void;
 };
 
+const resolver = {
+  Box,
+  Container,
+  TwoColumn,
+  Title,
+  Text,
+  Image,
+  WhiteSpace,
+  AboutReportWidget,
+  SurveyResultsWidget,
+  VisitorsWidget,
+  VisitorsTrafficSourcesWidget,
+  AgeWidget,
+  GenderWidget,
+  ProjectTemplate,
+};
+
 const Editor: React.FC<EditorProps> = ({
   onNodesChange,
   isPreview,
   children,
 }) => {
   return (
-    <BaseEditor
```

Comment:
Why have we removed the need for our own base editor?

Reply:
Because the 'Base Editor' used in the other content builder is just the craftjs editor with the render node plugged in. Since we're now even using our own render node there's no reason to use the old 'Base Editor' over the craftjs default one anymore